cumulusci.yml

```diff
@@ -825,6 +832,12 @@ flows:
                     api_names: QA,Gift_Entry
 
 
+    deploy_unmanaged:
```

Comment:
@davidmreed Can you confirm that the 'deploy_unmanaged` flow will only execute when deploying unmanaged npsp into an org - one that doesn't have a namespace?

Reply:
No. Deploy unmanaged will also run for deployments into namespaced scratch orgs.

Reply:
Glad I asked! Is there a flow/task that I can hook into here that will **only** deploy into an unmanaged org (including not deploying into the packaging org)?

Reply:
Though, now that I think of it, that might actually be good as is. My goal is for the class in this PR to never be in the packaging org. If it exists in a scratch org with a namespace, that probably would be ok.

Reply:
You could include a task in `deploy_unmanaged` with a `when` clause, like `when: not org_config.namespace`. I think that'd work fine.

Reply:
Good idea. Testing that now.

Reply:
Worked perfectly, thank you!! 
I'm still finalizing the method signatures for the `sfdoLogUtils` class in core, but ideally this will be ready for merging in the next couple of weeks.